pix-editor/app/components/form/whitelisted-url.js

```diff
@@ -64,7 +64,7 @@ export default class WhitelistedUrlForm extends Component {
     try {
       await this.args.onFormSubmitted(formData);
     } catch (err) {
-      this.errorMessages.pushObjects(err.message.split('\n'));
+      this.errorMessages.push(err.message.split('\n'));
```

Comment:
```suggestion
      this.errorMessages.push(...err.message.split('\n'));
```
Oubli de l'opérateur de tartinage ici